apps/sleeperfantasyfootball/sleeperfantasynfl.star

```diff
@@ -0,0 +1,688 @@
+                        cross_align = "start",
+                        children = [
+                            render.Column(
+                                children = output,
+                            ),
+                        ],
+                    ),
+                ],
+            )
+
+        else:
+            team_records = get_team_records(user_and_roster_map, league_rosters)
+            sorted_team_winning_percentage = get_team_winning_percentage(user_and_roster_map, league_rosters)
+            entries_to_display = teams_per_view
+            standings = calculate_standings(sorted_team_winning_percentage, team_records)
+            for x in range(0, len(standings), entries_to_display):
+                render_category.extend(
+                    [
+                        render.Column(
+                            children = render_standings_and_records(x, standings, entries_to_display, heading_font_color, color_scheme, league_name, show_ties),
+                        ),
+                    ],
+                )
+    else:
+        entries_to_display = teams_per_view
+        league_name = "Sleeper Fantasy"
+
+        standings = [{"Name": "Dumpster Fire", "Standings": "1", "Wins": "10", "Losses": "4", "Ties": "0"}, {"Name": "Who is Mac Jones?", "Standings": "2", "Wins": "7", "Losses": "6", "Ties": "1"}, {"Name": "Campin my style", "Standings": "3", "Wins": "12", "Losses": "2", "Ties": "0"}, {"Name": "Mixon It Up", "Standings": "4", "Wins": "8", "Losses": "5", "Ties": "1"}, {"Name": "Lamar Ja’Marr & Dally G", "Standings": "5", "Wins": "7", "Losses": "7", "Ties": "0"}, {"Name": "I BEAT STEVE IN THE MARATHON", "Standings": "6", "Wins": "8", "Losses": "6", "Ties": "0"}, {"Name": "WelcomeToTheZappeParade", "Standings": "7", "Wins": "5", "Losses": "9", "Ties": "0"}, {"Name": "Amon-Ra-Ah-Ah-Ah", "Standings": "8", "Wins": "3", "Losses": "11", "Ties": "0"}, {"Name": "Mar-a-Lago Raiders", "Standings": "9", "Wins": "7", "Losses": "7", "Ties": "0"}, {"Name": "Everyday I'm Russell'n", "Standings": "10", "Wins": "6", "Losses": "8", "Ties": "0"}, {"Name": "Jeudy's Pontiac Bandits", "Standings": "11", "Wins": "4", "Losses": "10", "Ties": "0"}, {"Name": "Poppy's Belle and Dude Perfect", "Standings": "12", "Wins": "6", "Losses": "8", "Ties": "0"}]
+        for x in range(0, len(standings), entries_to_display):
+            render_category.extend(
+                [
+                    render.Column(
+                        children = render_standings_and_records(x, standings, entries_to_display, heading_font_color, color_scheme, league_name, show_ties),
+                    ),
+                ],
+            )
+
+    print("Main - End")
+    return render.Root(
+        delay = int(rotation_speed) * 1000,
+        show_full_animation = True,
+        child = render.Animation(children = render_category),
+    )
+
+def get_league_name(league_id):
+    league_name_cached = cache.get(league_id + "_league_name")
```

Comment:
Would be great to use the http module's built-in cache functionality. We may change the scope of the cache module down the road. Just pass `ttl_seconds` to `http.get()` to get going with it.